code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -142,14 +142,17 @@
 //Applies brute and burn damage to the organ. Returns 1 if the damage-icon states changed at all.
 //Damage will not exceed max_damage using this proc
 //Cannot apply negative damage
-/obj/item/bodypart/proc/receive_damage(brute = 0, burn = 0, stamina = 0, updating_health = TRUE, required_status = null)
+/obj/item/bodypart/proc/receive_damage(brute = 0, burn = 0, stamina = 0, blocked = FALSE, updating_health = TRUE, required_status = null)
+	var/hit_percent = (100-blocked)/100
```

Comment:
here blocked is an integer, but on L145 it's a boo

Reply:
Just copied how other code did it, fixed.